Software/scan_key/AuthenticationSQL.py

```diff
@@ -1,6 +1,7 @@
 from on_off import *
 from ErrorSQL import *
 import os
+import pygame
```

Comment:
To give a general answer to your question: So my idea was to have two functions. one that provides the keypad authentication if a  keypad is inserted and the other just for rfid.